wrappers/javascript/shared/src/indyVdr/IndyVdrRequest.ts

```diff
@@ -62,4 +64,8 @@ export class IndyVdrRequest {
   public free(): void {
     indyVdr.requestFree({ requestHandle: this.handle })
   }
+
+  public get __responseType__(): ResponseType | undefined {
+    return undefined
+  }
```

Comment:
```suggestion
  public readonly __responseType__!: ResponseType
```

Might be a bit cleaner as it also omits the `undefined` which would also remove the `NonNullable` the from above. I vaguely remember that this worked, might have missed something. If the suggestion does not work, I am 100% fine with your solution.

Reply:
Event though it is an internal type, it can still be accessed publicly. So reason making it | undefined is that this way the type is not lying as the value could be undefined. By setting that it will always be defined it can lead to runtime errors.